### ISSUE TITLE
Update noVNC link noVNC.com

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#829]
```
Please describe the issue:
The link to the noVNC project (In The Wild on the connectivity feature page).

The noVNC link should be updated either to 
http://kanaka.github.com/noVNC/noVNC/vnc.html or perhaps better to 
http://noVNC.com which redirects to the former but is the canonical online 
landing page for noVNC even if the page moves off github.

Please provide any additional information below.
The link to be updated is on this page: 
http://www.html5rocks.com/en/features/connectivity
```

Original issue reported on code.google.com by `buskan...@gmail.com` on 16 Feb 2012 at 5:06

**Comment by GoogleCodeExporter** [comment]
```
Also, the noVNC page has been updated to a newer version of noVNC with a 
different look so it would be good to update the thumbnail image.
```

Original comment by `buskan...@gmail.com` on 16 Feb 2012 at 5:07

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision cb06e13f3060.
```

Original comment by `ericbide...@html5rocks.com` on 2 Mar 2012 at 6:53
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 2 Mar 2012 at 6:53
- Added labels: **Milestone-Q12012**, **features**
